python/deployment/index.py

```diff
@@ -0,0 +1,31 @@
+from solcx import install_solc
+
+RESET = "\033[0m"
+BOLD = "\033[1m"
```

Comment:
The variables are used only once in the file, I do not see any profits from extracting them to the variables